### ISSUE TITLE
version 2.1 not available on Carthage

### ISSUE THREAD

**Issue by emadhegab** [ahoyconference/ahoyrtc-ios#5]
Hello .. I noticed that Version 2.1 is not yet on Carthage

**Comment by jortberends** [comment]
Hey @emadhegab, 
Experienced the same thing, looks like you'll need to use this URL: https://github.com/ahoyconference/ahoyrtc-ios/blob/master/CarthageAhoyRTC-bitcode.json
Version 2.1 is available there.

**Comment by emadhegab** [comment]
yeah I solve it like that.. but i can't find a reason behind the split ! specially that version 2.0 was having problems with submitting to appstore!

**Comment by jortberends** [comment]
AhoyRTC 2.0 indeed contains an error, which prevents AppStore Uploads.
